docs/root/install/tools/route_table_check_tool.rst

```diff
@@ -37,6 +37,9 @@ Usage
     -d,  --details
       Show detailed test execution results. The first line indicates the test name.
 
+    -o, --only-show-failures
```

Comment:
IMO we can drop the shorthand , since it doesn't relate well. We have ignored the shorthand for `disable-deprecation-check` previously.